### DISCUSSION TITLE
Integration with Next.js

### DISCUSSION THREAD

**Discussion by witmarr97** [mdbootstrap/TW-Elements#1533] [Share ideas & request features]
I hope you will release the React integration (I voted here #1443) but I'm not sure if you're planning to release something for Next.js, and that's what I really need :)

**Comment by filipkappa** [comment]
We've added the Next.js integration guide in our latest release!

You can learn more about it here:
https://tailwind-elements.com/docs/standard/integrations/next-integration/
